data/client/king_phisher/king-phisher-client.ui

```diff
@@ -2418,6 +2419,251 @@ Spencer McIntyre</property>
       </row>
     </data>
   </object>
+  <object class="GtkApplicationWindow" id="PluginManagerWindow">
+    <property name="width_request">550</property>
+    <property name="height_request">300</property>
+    <property name="can_focus">False</property>
+    <property name="border_width">5</property>
+    <property name="title" translatable="yes">Plugin Manager</property>
+    <property name="window_position">center-on-parent</property>
+    <property name="destroy_with_parent">True</property>
+    <child>
+      <object class="GtkBox" id="box29">
```

Comment:
As King Phisher progresses in the build, we should start naming to identify the sub-components of the windows.` id = "PluginManangerWindow.main_box"` . Just in case we need to manipulate or call the object in a future.